ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -670,14 +670,20 @@ queue ${ARTIFACT_DIR}/metrics/job_metrics.json oc --insecure-skip-tls-verify rsh
 
 wait
 
+mkdir -p ${ARTIFACT_DIR}/junit/
+
+if openshift-tests e2e-analysis --help &>/dev/null; then
+    echo "Post e2e-analysis check for the cluster"
+    openshift-tests e2e-analysis --junit-dir "${ARTIFACT_DIR}/junit" || true
```

Comment:
@neisw FYI, as what we discussed last time, the new tests generated by this new sub command will not replace the original `operator conditions XXX` tests, but add new tests like `verify operator conditions XXX`.

Reply:
Thank you @jianlinliu - we will eventually remove the original tests for >= 4.21 once we have seen these running for a while.